Client/sdk/game/CCarEnterExit.h

```diff
@@ -23,4 +23,6 @@ class CCarEnterExit
                                             bool bCheckIfRoomToGetIn) = 0;
     virtual int  ComputeTargetDoorToExit(CPed* pPed, CVehicle* pVehicle) = 0;
     virtual bool IsRoomForPedToLeaveCar(CVehicle* pVehicle, int iDoor, CVector* pUnknown = 0) = 0;
+
+    virtual void GetPositionToOpenCarDoor(CVector& position, CVehicle* vehicle, unsigned int door) const noexcept = 0;
```

Comment:
You have changed `door` to `uint32_t` in `CCarEnterExitSA` but it's still `unsigned int` here.

Reply:
Interesting, I must have accidentally undone it